CONTRIBUTING.md

```diff
@@ -299,6 +299,19 @@ Example:
 }
 ```
 
+Overrides specifically for CodeCatalyst can be set using the `aws.dev.codecatalystService` setting. This is a JSON object consisting of keys/values required to override API calls to CodeCatalyst: `region`, `endpoint`, `hostname`, and `gitHostname`. If this setting is present, then all keys need to be explicitly provided.
```

Comment:
What was the reason you needed more granularity? In my use cases it was nice and easy since I only had to switch the setting from `prod` to `gamma` but now will need to add in all this info manually.
Will be helpful if you indicate that in the PR description.

Reply:
Will update the PR description. Yes you do have to specify all the values in settings.json, but it is a one time setup. Since settings.json supports comments, you can comment out different configurations as needed.